pkg/api/v1/clients/kube/cache/cache_test.go

```diff
@@ -90,7 +109,6 @@ var _ = Describe("kube core cache tests", func() {
 			})
 
 			It("can list resources for all listers", func() {
-				Expect(cache.NamespaceLister()).To(BeNil())
```

Comment:
Can we update this check in the other direction now that we always initialize a namespace lister? I.E.,
`Expect(cache.NamespaceLister()).NotTo(BeNil())`